### PR TITLE
Restyle and fix Redirects

### PR DESCRIPTION
**Pull request by thomasdax98**
---

## PR Checklist

-   [x] Verify if the change requires a changeset. See [CONTRIBUTING.md](https://github.com/vivid-planet/comet/blob/HEAD/CONTRIBUTING.md)
-   [x] Link to the respective task if one exists: COM-935
-   [x] Provide screenshots/screencasts if the change contains visual changes

<details>
    <summary>Screenshots/screencasts</summary>
    

Grid:

Previously:

Bildschirmfoto 2024-07-16 um 10 58 40

Now:

Bildschirmfoto 2024-07-16 um 10 42 48

Form:

Previously:

Bildschirmfoto 2024-07-16 um 10 58 47

Now:

Bildschirmfoto 2024-07-16 um 10 42 04

</details>

### CODE REVIEW

**Comment by thomasdax98** [comment]
I still have to check if the redirect bug also exists in previous versions

File: packages/admin/cms-admin/src/redirects/RedirectForm.tsx [hunk 1]
```diff
@@ -112,13 +113,14 @@ export const RedirectForm = ({ mode, id, linkBlock, scope }: Props): JSX.Element
 
             const { data } = await client.query<GQLRedirectSourceAvailableQuery, GQLRedirectSourceAvailableQueryVariables>({
                 query: gql`
-                    query RedirectSourceAvailable($scope: RedirectScopeInput!, $source: String!) {
-                        redirectSourceAvailable(scope: $scope, source: $source)
+                    query RedirectSourceAvailable($scope: RedirectScopeInput!, $source: String!, $excludedId: ID) {
+                        redirectSourceAvailable(scope: $scope, source: $source, excludedId: $excludedId)
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/redirects/RedirectForm.tsx:117]: In the page tree we check this in the Admin, should we do this here as well?
https://github.com/vivid-planet/comet/blob/app-router-predefined-pages/packages/admin/cms-admin/src/pages/createEditPageNode.tsx#L149-L152

File: packages/admin/cms-admin/src/redirects/createRedirectsPage.tsx [hunk 2]
```diff
@@ -41,7 +42,8 @@ function createRedirectsPage({ customTargets, scopeParts = [] }: CreateRedirects
             <Stack topLevelTitle={intl.formatMessage({ id: "comet.pages.redirects", defaultMessage: "Redirects" })}>
                 <StackSwitch initialPage="redirectsTable">
                     <StackPage name="redirectsTable">
```
Comment (johnnyomair) [inline packages/admin/cms-admin/src/redirects/createRedirectsPage.tsx:44]:
```suggestion
                <StackSwitch initialPage="grid">
                    <StackPage name="grid">
```

This should be grid as well.

File: packages/admin/cms-admin/src/redirects/RedirectForm.tsx (hunk 1 above)
Comment (thomasdax98) [reply packages/admin/cms-admin/src/redirects/RedirectForm.tsx]: I just looked at the problem again and realised that I completely misunderstood it 😅

The described behavior only occurred on the add page. The real reason was that there was no redirect from the add page to the edit page after creating the redirect. Therefore, the validation failed. My initial "fix" actually only hid this problem.

Now I correctly redirect to the edit page after creation

File: .changeset/stale-pandas-watch.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+---
+"@comet/cms-admin": patch
+"@comet/cms-api": patch
```
Comment (johnnyomair) [inline .changeset/stale-pandas-watch.md:3]:
```suggestion
```

This is an Admin change only

File: .changeset/stale-pandas-watch.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+---
+
+Redirect to edit page after adding a redirect
+
+Previously, the use wasn't redirected to the edit page after creating a new redirect.
```
Comment (johnnyomair) [inline .changeset/stale-pandas-watch.md:8]:
```suggestion
Previously, the user wasn't redirected to the edit page after creating a new redirect.
```

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**